js/setupFirebase.js

```diff
@@ -15,7 +15,6 @@ firebase.analytics();
 firebase.auth().onAuthStateChanged(function (user) {
   if (user) {
     // User is signed in.
-    var displayName = user.displayName;
     var email = user.email;
```

Comment:
actually comment was not only for displayName but for all variables here. You're not using them.